components/media-feature/Component.jsx

```diff
@@ -66,6 +69,7 @@ export default function MediaFeature({
           // src="https://picsum.photos/1300/"
           src={bgImageData.url}
           className="su-absolute su-size-full su-object-cover su-left-0 su-top-0 su-z-[1]"
+          alt={bgImageData.attributes?.alt || ""}
```

Comment:
Add alt tag for the background image - if there's no alt text, add `alt=""` for a11y